presto-main/src/main/resources/webapp/src/components/QueryList.jsx

```diff
@@ -63,13 +63,25 @@ export class QueryListItem extends React.Component {
         }
 
         let formattedQueryText = "";
+        let currentLineCount = 0;
 
         for (let i = 0; i < lines.length; i++) {
             const trimmedLine = lines[i].substring(minLeadingWhitespace).replace(/\s+$/g, '');
+            const maxWidthPerLine = 75;
```

Comment:
nit - This should be outside of the for-loop given that the value doesn't change.

Reply:
same as maxLines which exists in the for loop?

Reply:
moved